src/index.js

```diff
@@ -201,7 +201,7 @@ class DropdownTreeSelect extends Component {
         }}
       >
         <div className="dropdown">
-          <a className={dropdownTriggerClassname} onClick={!this.props.disabled && this.handleClick}>
+          <a className={dropdownTriggerClassname} onClick={!this.props.disabled && this.handleClick} role="button" tabIndex="0" aria-haspopup="listbox" aria-expanded={this.state.showDropdown}>
```

Comment:
Noticed that you cannot trigger the click with keyboard (Enter, Space) when you're now able to tab to the control.
We probably should add a onKeyDown which checks if Enter or Space is entered which focuses the input field instead.
The default action onclick should be focus on the input imho also.

Then you show the dropdown as before but also can start typing right away to filter the results. When #225 is in place you also can navigate to nodes with keyboard.